src/blueprints/Do.Blueprints.Service.Application/ServiceSpecExtensions.cs

```diff
@@ -52,6 +53,34 @@ public static class ServiceSpecExtensions
 
     #endregion
 
+    #region Integer
+
+    public static int AnInteger(this Stubber _) => 42;
+
+    #endregion
+
+    #region MemoryCache
+
+    public static IMemoryCache AMemoryCache(this Stubber giveMe,
+        bool clear = false
+    )
+    {
+        var getMemoryCache = giveMe.The<Func<IMemoryCache>>();
+        var memoryCache = getMemoryCache();
+
+        if (clear)
+        {
+            (memoryCache as MemoryCache)?.Clear();
+        }
+
+        return memoryCache;
+    }
+
+    public static void VerifyCount(this IMemoryCache memoryCache, Func<int, bool> condition) =>

```

Comment:
i see no active requirement for accepting a function parameter. let's make it simpler and accept `int`